fixtures/american-dj/encore-lp12z-ip.json

```
@@ -0,0 +1,580 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Encore LP12Z IP",
+  "shortName": "encore-lp12z-ip",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Nikita"],
+    "createDate": "2024-09-10",
+    "lastModifyDate": "2024-09-10"
+  },
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/13583/ADJ%20Encore%20LP12Z%20IP%20-%20User%20Manual%2002-22-2024.pdf",
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/13583/ADJ%20Encore%20LP12Z%20IP%20-%20DMX%20Traits.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/encore-lp12z-ip"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=f_LbX1cYPYw",
+      "https://www.youtube.com/shorts/7YWwZOtYoiw",
+      "https://www.youtube.com/shorts/4KwpsqBNHNc"
+    ]
+  },
+  "physical": {
+    "dimensions": [371, 316, 346],
+    "weight": 7.5,
+    "power": 227,
+    "DMXconnector": "3-pin XLR IP65",
+    "bulb": {
+      "type": "LED",
+      "lumens": 4400
+    },
+    "lens": {
+      "degreesMinMax": [5, 30]
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "fineChannelAliases": ["Red fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Green": {
+      "fineChannelAliases": ["Green fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Blue": {
+      "fineChannelAliases": ["Blue fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Lime": {
+      "fineChannelAliases": ["Lime fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Lime",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Color Macros": {
+      "capability": {
+        "type": "NoFunction"
+      }
+    },
+    "Shutter / Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 31],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed"
+        },
+        {
+          "dmxRange": [32, 63],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [64, 95],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [96, 127],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [128, 159],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Pulse",
+          "comment": "Pulse Effect in Sequences"
```

"Pulse" doesn't need to be repeated

```suggestion
          "comment": "in sequences"
```